patches/aspnetcore/0016-remove-temp-feeds.patch

```diff
@@ -1,31 +1,32 @@
-From 5b51114f69120bfcdfc3b2cdbbd4d6b1eb4d6450 Mon Sep 17 00:00:00 2001
+From 8fd0dfe23c93597c5f6176ea7e1981fd66918739 Mon Sep 17 00:00:00 2001
 From: Aditya Aggarwal <aditya.aggarwal@microsoft.com>
 Date: Mon, 16 Mar 2020 04:41:27 +0000
 Subject: [PATCH 16/16] remove temp feeds
 
 ---
- NuGet.config | 8 --------
- 1 file changed, 8 deletions(-)
+ NuGet.config | 9 ---------
+ 1 file changed, 9 deletions(-)
 
 diff --git a/NuGet.config b/NuGet.config
-index 9b63a8f022..cea8f2fb85 100644
+index ee9b0f6393..cea8f2fb85 100644
 --- a/NuGet.config
 +++ b/NuGet.config
-@@ -2,14 +2,6 @@
+@@ -2,15 +2,6 @@
  <configuration>
    <packageSources>
      <clear />
 -    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
 -    <add key="darc-pub-dotnet-corefx-8a3ffed" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-8a3ffed5/nuget/v3/index.json" />
--    <add key="darc-int-dotnet-extensions-3b1f2b7" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-extensions-3b1f2b7c/nuget/v3/index.json" />
--    <add key="darc-int-dotnet-efcore-0e7e329" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-efcore-0e7e329a/nuget/v3/index.json" />
--    <add key="darc-int-dotnet-core-setup-0c2e69c" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-core-setup-0c2e69ca/nuget/v3/index.json" />
+-    <add key="darc-int-dotnet-extensions-877455f" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-extensions-877455ff/nuget/v3/index.json" />
+-    <add key="darc-int-dotnet-efcore-5f01ecc" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-efcore-5f01ecc8/nuget/v3/index.json" />
+-    <add key="darc-int-dotnet-core-setup-65cd789" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-core-setup-65cd7897/nuget/v3/index.json" />
+-    <add key="darc-int-dotnet-corefx-620cea9" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-corefx-620cea9c/nuget/v3/index.json" />
 -    <add key="darc-int-dotnet-corefx-059a4a1" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-corefx-059a4a19/nuget/v3/index.json" />
--    <add key="darc-int-dotnet-aspnetcore-tooling-a8242d7" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-aspnetcore-tooling-a8242d79/nuget/v3/index.json" />
+-    <add key="darc-int-dotnet-aspnetcore-tooling-0e01666" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-aspnetcore-tooling-0e016665/nuget/v3/index.json" />
 -    <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
```

Comment:
Is it possible to remove this patch entirely too, or am I missing something unusual here?

Reply:
Yes, think I just missed that one in the merge, thanks.  Removed.